src/models/RSVPModel.js

```diff
@@ -0,0 +1,57 @@
+const mongoose = require('mongoose')
+
+const RSVP = mongoose.model('RSVP', {
```

Comment:
Think about how we should expire out RSVP, ex if user is accepted but RSVPS after 7 days how do we prevent them from creating a document